src/main/java/core/basesyntax/FigurePrinter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigurePrinter {
+    void printFigure();
+}

```

Comment:
Don't give to interfaces concrete names, better to have something abstract like `Printer` or `Drawable` or `Paintable`

https://howtodoinjava.com/java/basics/java-naming-conventions/